sys/net/crosslayer/ng_pktbuf/ng_pktbuf.c

```diff
@@ -23,11 +23,10 @@
 
 #include "clist.h"
 #include "mutex.h"
-#include "net/ng_nettype.h"
-#include "net/ng_pkt.h"
 #include "utlist.h"
-
 #include "net/ng_pktbuf.h"
+#include "net/ng_nettype.h"
+#include "net/ng_pkt.h"
```

Comment:
Nitpicky, but I would leave the `#include "net/ng_pktbuf.h"` at the last. You decide.